### PR TITLE
relnotes-iteration1

### PR DESCRIPTION
**Pull request by kaarthis**
Please leave comments or approve.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-03
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* OSsku=cblmariner (Mariner 1.0) reached end of life on July 31, and is no longer receiving security updates, in accordance with Mariner's 2 year lifecycle for major versions. Please migrate to OSsku=azurelinux (Mariner 2.0). Read more on [Mariner 2.0][https://microsoft.github.io/CBL-Mariner/announcing-mariner-2.0/]
```
Comment (allyford) [inline CHANGELOG.md:10]: Might be removed, waiting on azlinux team

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023
```
Comment (allyford) [inline CHANGELOG.md:11]: Since sept 2 is already passed, should we update to say that we did disable it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP][https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress] can now be a combination of ip/ipprefix and managed ones.
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference][https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797]
+  * Now customers can disable OutboundNAT for Windows nodes as long as the cluster's outbound type is not Load Balancer. This [change][https://learn.microsoft.com/en-us/azure/aks/nat-gateway#disable-outboundnat-for-windows-preview] enables customers to DisableOutboundNAT in conjunction with User Defined Routes (UDR) and Azure firewall. Before the modification, customers could only disable OutboundNAT for Windows nodes when the cluster's outbound type was NAT Gateway.
```
Comment (allyford) [inline CHANGELOG.md:19]:
```suggestion
  * Now customers can disable OutboundNAT for Windows nodes as long as the cluster's outbound type is not Load Balancer. This [change][https://learn.microsoft.com/azure/aks/nat-gateway#disable-outboundnat-for-windows-preview] enables customers to DisableOutboundNAT in conjunction with User Defined Routes (UDR) and Azure firewall. Before the modification, customers could only disable OutboundNAT for Windows nodes when the cluster's outbound type was NAT Gateway. 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP][https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress] can now be a combination of ip/ipprefix and managed ones.
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference][https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797]
+  * Now customers can disable OutboundNAT for Windows nodes as long as the cluster's outbound type is not Load Balancer. This [change][https://learn.microsoft.com/en-us/azure/aks/nat-gateway#disable-outboundnat-for-windows-preview] enables customers to DisableOutboundNAT in conjunction with User Defined Routes (UDR) and Azure firewall. Before the modification, customers could only disable OutboundNAT for Windows nodes when the cluster's outbound type was NAT Gateway. 
+  * Updated Windows gmsa[https://learn.microsoft.com/en-us/azure/aks/use-group-managed-service-accounts] webhook to v0.7.1[https://github.com/kubernetes-sigs/windows-gmsa/tree/v0.7.1] which supports multi-arch (amd64 and arm64).
```
Comment (allyford) [inline CHANGELOG.md:20]:
```suggestion
  * Updated Windows gmsa[https://learn.microsoft.com/azure/aks/use-group-managed-service-accounts] webhook to v0.7.1[https://github.com/kubernetes-sigs/windows-gmsa/tree/v0.7.1] which supports multi-arch (amd64 and arm64).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023
+on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP][https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress] can now be a combination of ip/ipprefix and managed ones.
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference][https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797]
```
Comment (allyford) [inline CHANGELOG.md:18]: * GMSA can now be disabled on an existing cluster. https://learn.microsoft.com/azure/aks/use-group-managed-service-accounts

File: CHANGELOG.md (hunk 2 above)
Comment (CocoWang-wql) [inline CHANGELOG.md:11]: Any link?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,31 @@
...
+* OSsku=cblmariner (Mariner 1.0) reached end of life on July 31, and is no longer receiving security updates, in accordance with Mariner's 2 year lifecycle for major versions. Please migrate to OSsku=azurelinux (Mariner 2.0). Read more on [Mariner 2.0][https://microsoft.github.io/CBL-Mariner/announcing-mariner-2.0/]
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023
+on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes
+
```
Comment (phealy) [inline CHANGELOG.md:15]: Where is the 1.28 preview?

File: CHANGELOG.md (hunk 1 above)
Comment (PixelRobots) [reply CHANGELOG.md:10]: Can we fix the link [ ] should be ( )

File: CHANGELOG.md (hunk 2 above)
Comment (kaarthis) [reply CHANGELOG.md]: This was broken i mended the message its from a prior comms we gave in last two rel notes too so no changes from me.

File: CHANGELOG.md (hunk 6 above)
Comment (kaarthis) [reply CHANGELOG.md:15]: Added that thanks to your sleuthing this morning :)

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,32 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-03
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* AKS 1.28 preview [version](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#v1280) is now available.
```
Comment (palma21) [inline CHANGELOG.md:10]: Why is this in announcements instead of Preview features?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,32 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023, on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch or NodeImage options](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+* [Node OS Upgrade Channel - NodeImage](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) is now generally available.
```
Comment (palma21) [inline CHANGELOG.md:12]: Same why is this in announcements instead of Features

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,32 @@
...
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress) can now be a combination of ip/ipprefix and managed ones.
```
Comment (palma21) [inline CHANGELOG.md:17]: Why is this a behavioral change instead of a feature?

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,32 @@
...
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference](https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797).
```
Comment (palma21) [inline CHANGELOG.md:18]: What does this mean? And what change in behavior does it imply?

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,32 @@
...
+  * Now customers can disable OutboundNAT for Windows nodes as long as the cluster's outbound type is not Load Balancer. This [change](https://learn.microsoft.com/azure/aks/nat-gateway#disable-outboundnat-for-windows-preview) enables customers to disable OutboundNAT in conjunction with User Defined Routes (UDR) and Azure firewall. Before the modification, customers could only disable OutboundNAT for Windows nodes when the cluster's outbound type was NAT Gateway.
```
Comment (palma21) [inline CHANGELOG.md:19]: Why is this not a feature?

Comment (palma21) [reply CHANGELOG.md:19]: Also is this GA?

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,32 @@
...
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress) can now be a combination of ip/ipprefix and managed ones.
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference](https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797).
+  * Now customers can disable OutboundNAT for Windows nodes as long as the cluster's outbound type is not Load Balancer. This [change](https://learn.microsoft.com/azure/aks/nat-gateway#disable-outboundnat-for-windows-preview) enables customers to disable OutboundNAT in conjunction with User Defined Routes (UDR) and Azure firewall. Before the modification, customers could only disable OutboundNAT for Windows nodes when the cluster's outbound type was NAT Gateway. 
+  * Updated Windows [gmsa](https://learn.microsoft.com/azure/aks/use-group-managed-service-accounts) webhook to [v0.7.1](https://github.com/kubernetes-sigs/windows-gmsa/tree/v0.7.1) which supports multi-arch (amd64 and arm64).
```
Comment (palma21) [inline CHANGELOG.md:20]: Why is this not a component update?

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,32 @@
...
+  * The taint added by AKS remediator will change from `remediator.aks.microsoft.com/unschedulable` to `remediator.kubernetes.azure.com/unschedulable`.
```
Comment (palma21) [inline CHANGELOG.md:21]: Link to node auto repair. That is also the public name

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 11 above)
Comment (kaarthis) [reply CHANGELOG.md:19]: it says preview the doc so i ll prob put in preview unless @allyford says GA.

**Review by CocoWang-wql — APPROVED** [review]

**Review by aritraghosh — APPROVED** [review]

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,32 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023, on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch or NodeImage options](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes
+* Preview Features
+  * AKS 1.28 preview [version](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#v1280) is now available.
```
Comment (palma21) [inline CHANGELOG.md:14]: [Kubernetes 1.28](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#v1280) is now available in preview.

**Review by palma21 — APPROVED** [review]

**Pull request merged**